fix(forms): Phone number merged with country code, bugfixes

henit opened the pull request.

tujoworker commented on packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/feature-fields/PhoneNumber/properties.mdx:
```diff
@@ -15,6 +15,7 @@ import DataValueReadwriteProperties from '../../data-value-readwrite-properties.
 | `help`                                  | `object`            | _(optional)_ Provide a help button. Object consisting of `title` and `contents`                             |
 | `countryCodeFieldClassName`             | `string`            | _(optional)_ Class name for the country code component.                                                     |
 | `numberFieldClassName`                  | `string`            | _(optional)_ Class name for the number component.                                                           |
+| `counttryCodePlaceholder`               | `string`            | _(optional)_ Placeholder for the country code field.                                                        |
```
```suggestion
| `countryCodePlaceholder`               | `string`            | _(optional)_ Placeholder for the country code field.                                                        |
```

langz replied:
```suggestion
| `countryCodePlaceholder`               | `string`            | _(optional)_ Placeholder for the country code field.                                                        |
```

tujoworker commented on packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/PhoneNumber.tsx:
```diff
@@ -13,6 +14,7 @@ export type Props = FieldHelpProps &
   FieldProps<string, undefined> & {
     countryCodeFieldClassName?: string
     numberFieldClassName?: string
+    counttryCodePlaceholder?: string
```
```suggestion
    countryCodePlaceholder?: string
```

tujoworker commented on packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/style/dnb-phone-number.scss:
```diff
@@ -1,20 +1,8 @@
-
   &__country-code {
-    flex: 0 9rem;
+    width: 9rem;
```
Now, we have used `9rem` before in this PR, but is does it have a relation to one of these field sizes? `--forms-field-width--medium` – could 

```scss
calc(var(--forms-field-width--medium) - 2rem) // with a comment why -2rem
```

do the same?

tujoworker reviewed (commented):
Nice work! I just have some comments to outline as first step.

tujoworker commented:
Also, there is a conflict with `CountryCode.tsx` in this PR.

langz commented:
I added a commit, https://github.com/dnbexperience/eufemia/pull/2755/commits/25d8ca505d2b7aab84d9700a5512b24cb859b4d7,  renaming `counttryCodePlaceholder` to `countryCodePlaceholder`.

henit commented:
> I added a commit, [25d8ca5](https://github.com/dnbexperience/eufemia/commit/25d8ca505d2b7aab84d9700a5512b24cb859b4d7), renaming `counttryCodePlaceholder` to `countryCodePlaceholder`.

Ah, nice. I added that one to check if you where paying attention 🤣

tujoworker commented:
:tada: This PR is included in version 10.10.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@dnb/eufemia/v/10.10.0)
- [GitHub release](https://github.com/dnbexperience/eufemia/releases/tag/v10.10.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
